pyccel/codegen/printing/fcode.py

```diff
@@ -79,22 +78,78 @@
 __all__ = ["FCodePrinter", "fcode"]
 
 known_functions = {
-    "sin": "sin",
-    "cos": "cos",
-    "tan": "tan",
-    "asin": "asin",
-    "acos": "acos",
-    "atan": "atan",
-    "atan2": "atan2",
-    "sinh": "sinh",
-    "cosh": "cosh",
-    "tanh": "tanh",
-    "log": "log",
-    "exp": "exp",
-    "erf": "erf",
-    "Abs": "abs",
-    "sign": "sign",
-    "conjugate": "conjg"
+    "sign": "sign",       # TODO: move to numpyext
+    "conjugate": "conjg"  # TODO: move to numpyext
+}
+
+numpy_ufunc_to_fortran = {
+    'NumpyAbs'  : 'abs',
+    'NumpyFloor': 'floor',  # TODO: might require special treatment with casting
+    # ---
+    'NumpyExp' : 'exp',
+    'NumpyLog' : 'Log',
+    'NumpySqrt': 'Sqrt',
```

Comment:
Since this dictionary entry is not used (`sqrt` is a special case), for the sake of clarity I suggest to comment this line out and add a comment.